test/functional/p2p-acceptblock.py

```diff
@@ -144,7 +144,7 @@ def run_test(self):
                 assert_equal(x['status'], "headers-only")
                 tip_entry_found = True
         assert(tip_entry_found)
-        assert_raises_rpc_error(-1, "Block not found on disk", self.nodes[0].getblock, block_h2f.hash)
+        self.nodes[0].getblock(block_h2f.hash)
```

Comment:
If you're going to squash this in to `Rewrite p2p-acceptblock in preparation for slight behavior changes`, then you'll need to have this line commented out (and re-uncomment it in `Accept unrequested blocks with work equal to our tip`), in order to make the test succeed after that first commit, I think.

Reply:
Oh ffs, I'm just gonna remove that line.